notebooks/DrizzlePac/drizzle_wfpc2/drizzle_wfpc2.ipynb

```diff
@@ -18,7 +18,28 @@
    "cell_type": "markdown",
```

Comment:
<p>Also, there seems to be an issue with the link to <code>'Making WFPC2 Images Compatible with AstroDrizzle'</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/137/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>